Sparkle/SUBinaryDeltaCreate.m

```diff
@@ -138,11 +220,25 @@ int createBinaryDelta(NSString *source, NSString *destination, NSString *patchFi
         }
 
         NSDictionary *info = infoForFile(ent);
+        if (!info) {
+            fprintf(stderr, "Failed to retrieve info for file %s", ent->fts_path);
+            return 1;
+        }
         originalTreeState[key] = info;
+        
+        if (aclExists(ent)) {
```

Comment:
That looks like an unrecoverable error. Maybe it's better to ignore or override ACLs?